docs/report/main.tex

```diff
@@ -23,9 +23,21 @@
 \usepackage{mathtools}
 \usepackage{physics}
 \usepackage{amsmath}
-
+\usepackage[superscript,biblabel]{cite}
 \usepackage[scale=1,bmargin=1.5cm,footnotesep=1.5cm]{geometry}
 
+\usepackage{hyperref}
+\hypersetup{
+    colorlinks=true,
+    linkcolor=violet,
+    filecolor=green,      
+    urlcolor=blue,
+    pdftitle={Overleaf Example},
```

Comment:
Should this be modified to "Quantum Computing Project" or something along those lines?

Reply:
Good shout, will change this